app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,35 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = logged_user(session_params)
+    if user
+      session[:user_id] = user.id
+      redirect_to root_path
+      flash[:success] = "Welcome to forsythia!"
+    else
+      redirect_to '/login'
+      flash[:error] = "Sorry, but your email or password were wrong."
+    end
+  end
+
+  def destroy
+    session[:user_id] = nil
+    redirect_to root_path
+  end
+
+  private
+
+  def session_params
+    params.require(:session).permit(:email, :password)
+  end
+
+  def logged_user(session_params)
+    user = User.find_by_email(session_params[:email])
+    if user && user.authenticate(session_params[:password])
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.](https://www.codacy.com/app/conta-srdr/forsythia/file/2131716925/issues/source?bid=3139784&fileBranchId=3147671#l30)